### PR TITLE
Highest/lowest total score and biggest blowout methods mr

### PR DESCRIPTION
**Pull request by mronauli**

### CODE REVIEW

File: lib/game_collection.rb [hunk 1]
```diff
@@ -13,4 +13,33 @@ def create_games(file_path)
 
     csv.map { |row| Game.new(row) }
   end
+
+  def highest_total_score
+    # highest_score = 0
```
Comment (tylertomlinson) [inline lib/game_collection.rb]: I don't know if its best to merged commented out code.

File: lib/game.rb [hunk 2]
```diff
@@ -6,4 +6,14 @@ def initialize(game_info)
     @away_goals = game_info[:away_goals].to_i
     @home_goals = game_info[:home_goals].to_i
   end
+
+  def total_score
```
Comment (d-atkins) [inline lib/game.rb:10]: This is really helpful

Comment (tylertomlinson) [inline lib/game.rb:10]: Great methods that will be useful!

File: test/game_collection_test.rb [hunk 3]
```diff
@@ -21,4 +21,16 @@ def test_it_can_create_games_from_csv
     assert_equal 2, @game.away_goals
     assert_equal 3, @game.home_goals
   end
+
+  def test_it_can_get_the_sum_of_highest_winning_and_losing_team_score
```
Comment (tylertomlinson) [inline test/game_collection_test.rb:25]: These needs to be fixed

Comment (Yetidancer) [reply test/game_collection_test.rb:25]: fixed

**Pull request merged**